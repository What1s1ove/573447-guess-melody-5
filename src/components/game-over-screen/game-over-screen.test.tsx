import * as React from 'react';
import { render } from 'enzyme';
import GameOverScreen from './game-over-screen';

describe(`GameOverScreen component`, () => {
  it(`should render GameOverScreen component`, () => {
    const component = render(<GameOverScreen />);

    expect(component).toMatchSnapshot();
  });
});
