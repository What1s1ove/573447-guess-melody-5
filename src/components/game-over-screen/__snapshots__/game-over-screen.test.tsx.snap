// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameOverScreen component should render GameOverScreen component 1`] = `
<section
  class="result"
>
  <div
    class="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    class="result__title"
  >
    Какая жалость!
  </h2>
  <p
    class="result__total result__total--fail"
  >
    У вас закончились все попытки. Ничего, повезёт в следующий раз!
  </p>
  <button
    class="replay"
    type="button"
  >
    Попробовать ещё раз
  </button>
</section>
`;
